administration/src/cards/extensions/extensions.tsx

```diff
@@ -0,0 +1,23 @@
+import { PartialMessage } from '@bufbuild/protobuf'
+
+import { CardExtensions } from '../../generated/card_pb'
+import { BavariaCardTypeExtension } from './BavariaCardTypeExtension'
+import { BirthdayExtension } from './BirthdayExtension'
+import { NuernbergPassNumberExtension } from './NuernbergPassNumberExtension'
+import { RegionExtension } from './RegionExtension'
+
+export abstract class Extension<T, R> {
+  public state: T | null = null
+  abstract setInitialState(args: R): void
+  abstract isValid(): boolean
+  abstract createForm(onChange: () => void): React.ReactElement | null
```

Comment:
i think its better to import the `ReactElement from react` explicitly instead importing the whole library